exercises/jigsaw-puzzle/instructions.md

```diff
@@ -0,0 +1,22 @@
+You get part of the information and need to fills in the blanks.
+In case the data is insufficient or contradictory the user needs to be notified of the issue.
+The full information about the jigsaw puzzle contains the following parts:
+
+- `pieces`: Number of pieces
+- `border`: Number of border pieces
+- `inside`: Number of inside pieces
+- `rows`: Number of rows of pieces
+- `columns`: Number of columns of pieces
+- `aspectRatio`: Aspect ratio of columns / rows
+- `format`: Puzzle format, which can be `portrait`, `square`, `landscape`, or `globe`
+
+For a puzzle in `globe` format, assume an aspect ratio of 1.0.
```

Comment:
Wouldn't a globe have twice as many columns as rows?

Reply:
That depends on the projection.